doc/manpages/generate-man-pages.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
+
+## Gen man-pages in meson builds
+
+# Why dinit dont use "env" argument in "run_command()"?
+# Because old meson (>0.50.0) dont support "env" in "run_command()".
```

Comment:
I think you mean `<0.50.0`. But also, why bother supporting old meson versions?

Reply:
Because; Newer Mesons don't have a MAJOR feature and don't have something that interest me.
I think it's ok! no?

Reply:
Doesn't seem ideal if it means we have to sprinkle the source tree with extra shell scripts, but ok

Reply:
Maybe I'm totally dumb here, but... why does this need env support at all?

```meson
foreach man: ['dinit.8', 'dinitcheck.8', 'dinitctl.8', 'dinit-monitor.8', 'dinit-service.5', 'shutdown.8']
    section = 'man' + man.split('.')[1]
    custom_target(
        man,
        input: man + '.m4',
        output: man,
        command: [m4, '-DVERSION='+version, '-DMONTH='+month, '-DYEAR='+year, '@INPUT@'],
        capture: true,
        install: true,
        install_dir: join_paths(get_option('mandir'), section),
    )

endforeach
```

Running it as a custom_target has the additional advantage that if you git pull and these man page sources are updated, they actually get rebuilt...